### ISSUE TITLE
Bug report: dmftproj cannot start with website example with WIEN2k 19.2

### ISSUE THREAD

**Issue by 70akaline** [TRIQS/dft_tools#153]
### Prerequisites

Indeed, there's an issue was closed w/o solution, even with the new version of Wien2k (19.2). It's still reproduced.
https://github.com/TRIQS/dft_tools/issues/122

### Description

I'm currently using Wien2k 19.2, after checking the dft_tool/triqs compilation the problem is still here. 
Following this issue #127, I tried to recompile lapw2 and executed `x lapw2 -almd` again.

Also, I removed all of the comments from website example file `case.indmftpr`

### Steps to Reproduce

Basically following the website example, 

1. Creat the `sturct` file, and run the initial SCF calculation. 
2. Then got the following `case.struct` file:
```plain
SrVO3                                                                          
P   LATTICE,NONEQUIV.ATOMS:  3221_Pm-3m                                        
MODE OF CALC=RELA unit=bohr                                                    
  7.261300  7.261300  7.261300 90.000000 90.000000 90.000000                   
ATOM   1: X=0.00000000 Y=0.00000000 Z=0.00000000
          MULT= 1          ISPLIT= 2
Sr         NPT=  781  R0=0.00001000 RMT=   2.50000   Z: 38.0                   
LOCAL ROT MATRIX:    1.0000000 0.0000000 0.0000000
                     0.0000000 1.0000000 0.0000000
                     0.0000000 0.0000000 1.0000000
ATOM   2: X=0.50000000 Y=0.50000000 Z=0.50000000
          MULT= 1          ISPLIT= 2
V          NPT=  781  R0=0.00005000 RMT=   1.91      Z: 23.0                   
LOCAL ROT MATRIX:    1.0000000 0.0000000 0.0000000
                     0.0000000 1.0000000 0.0000000
                     0.0000000 0.0000000 1.0000000
ATOM  -3: X=0.00000000 Y=0.50000000 Z=0.50000000
          MULT= 3          ISPLIT=-2
      -3: X=0.50000000 Y=0.00000000 Z=0.50000000
      -3: X=0.50000000 Y=0.50000000 Z=0.00000000
O          NPT=  781  R0=0.00010000 RMT=   1.70      Z:  8.0                   
LOCAL ROT MATRIX:    0.0000000 0.0000000 1.0000000
                     0.0000000 1.0000000 0.0000000
                    -1.0000000 0.0000000 0.0000000
  48      NUMBER OF SYMMETRY OPERATIONS
-1 0 0 0.00000000
.......... (see below attachment)
```

[02_srvo3.struct.txt](https://github.com/TRIQS/dft_tools/files/5598547/02_srvo3.struct.txt)

3. Executing `x lapw2 -almd`
4. Executing `dmftproj` with the following input file:
```plain
3
1 1 3
3
complex
1 0 0 0
0 0 0 0
cubic
1 1 2 0
0 0 2 0
01
0
complex
1 1 0 0
0 0 0 0
-0.11 0.14  0
```
Also attached below,
[02_srvo3.indmftpr.txt](https://github.com/TRIQS/dft_tools/files/5598549/02_srvo3.indmftpr.txt)

Then got the error message:
```log
~$ dmftproj
Welcome in DMFTPROJ: PROJECTION TO LOCALIZED BASIS


This prgm will build the Wannier projectors to the
localized orbitals of an atom onto which DMFT will be applied.


You are performing a computation
without Spin-Orbit.
using Paramagnetic Wien2k input files.
=======================================
Sorts of atoms =   3
Equivalent sites per each sort: 1 1 3


-------------------------------------
For the sort  1 :
The orbital l= 0 is included.

-------------------------------------
For the sort  2 :
The orbital l= 0 is included.
The orbital l= 1 is included.
The orbital l= 2 is included.

-------------------------------------
For the sort  3 :
The orbital l= 0 is included.
The orbital l= 1 is included.

=======================================
Basis representation for each sort.

At line 140 of file /home/physics/phrkjq/triqsdir/dft_tools.src/fortran/dmftproj/set_ang_trans.f
Fortran runtime error: Bad real number in item 1 of list input

Error termination. Backtrace:
#0  0x2afef86b3798 in read_real
	at ../../../libgfortran/io/list_read.c:2026
#1  0x2afef86b5b7d in list_formatted_read_scalar
	at ../../../libgfortran/io/list_read.c:2180
#2  0x40fe89 in ???
#3  0x4079f0 in ???
#4  0x40177c in ???
#5  0x310081ed1f in ???
#6  0x4017ac in ???
```

5. Further checked with the file `case.almblm`, the initial information looks like similar to [here](https://github.com/TRIQS/dft_tools/issues/127#issuecomment-575614154)
My `almblm` file initial part is:
```plain
   40.999999999899998     
         165
           3
  0.59735828992105056     
   7.0914314401440123E-002
           1
  0.34367101169150510      -0.23975354139745386     
  0.24782106354167119     
           1
  0.71306587463906701      -0.34727281823432560     
   4.9126811469069331E-002
           0
   8.9508994762890724E-003
           0
```

**Expected behavior:** [What you expect to happen]

**Actual behavior:** [What actually happens]

### Versions

I'm using the 3.0 version of TRIQS, dmft_tool, Wien2k version is 19.2, GCC version is 8.3.0, OpenMPI version is 3.1.4,
python version is 3.7.3

This bug only occurred when using Wien2k 19.2. I also tried Wien2k 18.2, and `dmftproj` works.

**Comment by aichhorn** [comment]
Hi,
You did only part of the solution explained in #127. The one thing you did is to fix the printing of the almblm file. But there is also an additional header in the case.cf_d_eg_t2g file (and all the other files in SRC_templates). So you need to go to the SRC_templates directory of your Wien2k installation and remove the header line starting with #. dmftproj expects a number on the first line, and not some text, that is why the code crashes.

**Comment by Simba2805** [comment]
Dear all,

I am using WIEN2K_21.1. I am facing the same error while working with dmftproj.

When I run `x lapw2 -almd -p` it gives me an error as follows:

`  Summary of lapw2para:`
`   localhost	 user=2.225	 wallclock=1950.4`
`cp: cannot stat ‘.in.tmp’: No such file or directory`
`**  LAPW2 crashed!`
`2.152u 6.728s 0:30.03 29.5%	0+0k 0+4176io 0pf+0w`
`error: command   /home/19ts91r02/WIEN2K_21.1/lapw2para lapw2.def   failed`

As suggested I did all the things suggested in the relevant threads.
1. Removed header lines starting with  '#' in the SRC_templates directory.
2. Added a line in the `INTEGER :: lmax_to_dmft = 3` at the very beginning of the module file `qdmft.F`.
3. Commented out the line `228` in file `l2main.F` (The line reads as: `!write(6,*)'NAT,NUME,NKPT',NAT,NUME,NKPT-1`)
4. Recompiled WIEN2K. I tried to compile the `lapw2`  program only but it also showed me some errors. But as suggested, the `make_clean` doesn't work because there is no `make_clean` file in `SRC_lapw2` directory.

I am running WIEN2K in parallel mode. And it gives me `*.almblm` files as well. 

If I ignore the above error and proceed with `dmftproj` command, it returns me the following error:

`Reading of the file SrVO3.almblm`
`At line 570 of file /home/conda/feedstock_root/build_artifacts/triqs_dft_tools_1603114267154/work/fortran/dmftproj/dmftproj.f (unit = 9, file = 'SrVO3.almblm')`
`Fortran runtime error: End of file`
`Error termination. Backtrace:`
`#0  0x2ab1d766b201 in list_formatted_read_scalar`
`	at ../../../libgfortran/io/list_read.c:2309`
` #1  0x55bdcbe740b6 in ???`
` #2  0x55bdcbe70070 in ???`
` #3  0x2ab1d9b86494 in ???`
` #4  0x55bdcbe700a0 in ???`

kindly help me sort out the issues.

**Comment by harrisonlabollita** [comment]
Hi,

The program `dmftproj` and wien2k routine `x lapw2 -almd` must be run in serial mode. This should fix the errors you are experiencing in both the `lapw2` routine and `dmftproj`. 

Best, 
Harry

**Comment by aichhorn** [comment]
Hi,

There is a possibility to run lapw2 -almd in parallel mode. However, this is not part of the public package, since it needs some changes to the Wien2k routines. If you want and need to do this, you can contact Leonid Pourovskii from Ecole Polytechnique. dmftproj is always done in serial mode, there is no performance issue there.

Best,
Markus

**Comment by the-hampel** [comment]
I think the issue is thereby addressed (wien2k and dmftproj need to be run in serial by default), so I will close the issue for now.
